docs/rancher/csi-driver.md

```diff
@@ -143,6 +143,34 @@ The only difference is in generating the `cloud-init` config where you need to s
 ./generate_addon_csi.sh <serviceaccount name> <namespace> k3s
 ```
 
+## Customize the default StorageClass
+
+Harvester CSI Driver provides the interface to pre-define the default StorageClass. If we do not specify the default StorageClass, the default StorageClass of Harvester CSI Driver will be the same as Harvester cluster.
+
+To customize the default StorageClass, you can use the parameter `host-storage-class`. Please check with the following example:
+
+1. Create new a StorageClass of the host Harvester cluster
+
+  ![](/img/v1.3/rancher/create-new-sc.png)
+
+2. Deploy CSI driver with parameter `host-storage-class`
+
+  ![](/img/v1.3/rancher/deploy-csi-driver-with-host-storage-class.png)
+
+Then, make sure the Harvester CSI driver is ready. You can verify with the following steps.
+
+1. Create volume with harvester CSI driver
+
+  ![](/img/v1.3/rancher/create-volume-with-harvester-csi-driver.png)
+
+2. Check the volume status `bound` and its PVC name
+  
+    ![](/img/v1.3/rancher/check-volume-and-pvc-name.png)
+
+3. Check the PVC name on the host harvester volume page. Then you can see it is created by the StorageClass you configured.
+
+  ![](/img/v1.3/rancher/check-pvc-name-on-host-harvester-volume-page.png)
```

Comment:
```suggestion
    3. On the **Volumes** screen, verify that the volume was provisioned using the StorageClass that you created.
    
        Example:
        ![](/img/v1.3/rancher/check-pvc-name-on-host-harvester-volume-page.png)
```

Reply:
I changed the description to match what we expect users to do on the UI. In this case, they must match the volume name and StorageClass name.